packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html

```diff
@@ -1,4 +1,7 @@
-<oui-spinner data-size="s" data-ng-if="$ctrl.isLoading"></oui-spinner>
+<oui-spinner
+    data-size="s"
+    data-ng-if="$ctrl.isLoading && $ctrl.currentStep < 2"
```

Comment:
Pretty sure `currentStep` does not exist on this controller, if you want to control the display of the spinner from outside the component, you should create a property

Reply:
Yes, I agree with @y4nnL, the components cannot just behave for the case of Instance creation.

Reply:
Done